README.md

```diff
@@ -131,6 +184,19 @@ Please see [Quickstarts](quickstart.md)
     azd down
     ```
 
+## Guidance
+
+### Region Availability
+
+This template uses gpt-4o and text-embedding-3-large models which may not be available in all Azure regions. Check for [up-to-date region availability](https://learn.microsoft.com/azure/ai-services/openai/concepts/models#standard-deployment-model-availability) and select a region during deployment accordingly
+  * We recommend using `eastus2', 'eastus', 'japaneast', 'uksouth', 'northeurope', or 'westus3'
+
+### Costs
+
+You can estimate the cost of this project's architecture with [Azure's pricing calculator](https://azure.microsoft.com/pricing/calculator/)
```

Comment:
@markjbrown - Any suggestions for how to provide cost estimation guidance to user's of this repo? Do you happen to have this writen down elsewhere?

Reply:
The cost of this thing is:

Azure Cosmos DB Serverless is $0.25 per 1M RU/s. This is strictly pay per request. Users can do about 100K queries with 1M RU. With the usage you'd get for this sample Cosmos probably runs at less than $1 a month.

App Service is running on a B1 so $12.41 USD/month

For Azure OpenAI:
This thing deploys gpt-40 configured for 10K TPM. The smallest gpt-4 is for 1M PTU for $5/month input tokens and $15/month output tokens so I suppose $20/month? 
for Text-3-large it deploys for 5K TPM this is less than 0.1c 

So maybe you can include this:
Average Monthly Cost:
Azure Cosmos DB Serverless ($0.25 USD per 1M RU/s): $0.25
Azure App Service (B1 Plan): $12.41
Azure OpenAI (GPT-4o 1M input/output tokens): $20 (Sample uses 10K tokens)
Azure OpenAI (text-3-large): < $0.01 (Sample uses 5K tokens)